### DISCUSSION TITLE
CORS proxy configuration for Browser

### DISCUSSION THREAD

**Discussion by DustinBrett** [DustinBrett/daedalOS#104] [Ideas]
I could allow accessing more URL's if they went through one of the public CORS proxies. But they are not reliable and the list changes.

**Comment by hello-smile6** [comment]
You can use mine. It's never been down that I've ever seen.
https://cors-anywhere.9pfs.repl.co/

**Reply by DustinBrett** [reply]
Haha ya cool, I would have a list of a few. And maybe allow customization.

https://theoldnet.com/ would be another fun one to add

**Reply by hello-smile6** [reply]
Mine actually does work well. I can sometimes get faster downloads through my CORS proxy than directly for websites like the Internet Archive.

**Reply by hello-smile6** [reply]
Also, just because it uses Replit, that doesn't mean it won't be able to handle large amounts of users. I actually have a NodeBB forum set up using Replit.
https://nodebb.9pfs.repl.co/
